examples/resources/plugins/teleport-mattermost.toml

```diff
@@ -1,22 +1,23 @@
 # example mattermost configuration TOML file
 [teleport]
-auth-server = "example.com:3025"  # Teleport Auth Server GRPC API address
-client-key = "/var/lib/teleport/plugins/mattermost/auth.key" # Teleport GRPC client secret key
-client-crt = "/var/lib/teleport/plugins/mattermost/auth.crt" # Teleport GRPC client certificate
-root-cas = "/var/lib/teleport/plugins/mattermost/auth.cas"   # Teleport cluster CA certs
+auth_server = "example.com:3025"                             # Teleport Auth Server GRPC API address
+client_key = "/var/lib/teleport/plugins/mattermost/auth.key" # Teleport GRPC client secret key
+client_crt = "/var/lib/teleport/plugins/mattermost/auth.crt" # Teleport GRPC client certificate
+root_cas = "/var/lib/teleport/plugins/mattermost/auth.cas"   # Teleport cluster CA certs
 
 [mattermost]
-token = "api-token"              # Mattermost Bot OAuth token
-secret = "signing-secret-value"  # Mattermost API Signing Secret
-# team = "mattermost-team-name"        # Mattermost Team Name
-# url = "mattermost.example.com:8065"  # Mattermost URL. With Optional Port. 
-channel = "channel-name"         # Mattermost Channel name to post requests to
+url = "https://mattermost.example.com" # Mattermost Server URL
+team = "team-name"                     # Mattermsot team in which the channel resides.
+channel = "channel-name"               # Mattermost Channel name to post requests to
+token = "api-token"                    # Mattermost Bot OAuth token
+secret = "signing-secret-value"        # Mattermost API signing Secret
 
 [http]
-listen = ":8081"          # Mattermost interaction callback listener
-# https-key-file = "/var/lib/teleport/plugins/mattermost/server.key"  # TLS private key
-# https-cert-file = "/var/lib/teleport/plugins/mattermost/server.crt" # TLS certificate
+public_addr = "example.com" # URL on which callback server is accessible externally, e.g. [https://]teleport-mattermost.example.com
+# listen_addr = ":8081" # Network address in format [addr]:port on which callback server listens, e.g. 0.0.0.0:443
```

Comment:
Port at the end should match the port at the beginning

i.e. `# Network address in format [addr]:port on which callback server listens, e.g. 0.0.0.0:8081`